### ISSUE TITLE
bug in parser with staticlookup nodes

### ISSUE THREAD

**Issue by evilebottnawi** [glayzzle/php-parser#179]
Input:
```php
($a->b)::call();
```

Output:
```
 SyntaxError: Parse Error : syntax error, unexpected '::' (T_DOUBLE_COLON), expecting ';' on line 38
```

**Comment by evilebottnawi** [comment]
@ichiriac looks bug still exists in latest released version

**Comment by ichiriac** [comment]
It was fixed but not yet released - will be done in prerelease 5 - you can try out with the latest commit or from the online parser release on https://glayzzle.com/

**Comment by evilebottnawi** [comment]
@ichiriac :+1: Wait release

**Comment by ichiriac** [comment]
released under 3.0.0-prerelease5
